Dockerfile

```diff
@@ -4,6 +4,9 @@ WORKDIR /package
 
 COPY . ./
 
+# Helps with: https://bugs.swift.org/browse/SR-6500
+RUN rm -rf /package/.build/debug
```

Comment:
this seems to have fixed the symlink error we get when run tests on linux+docker.